.github/workflows/release-create-kpt-functions.yaml

```diff
@@ -1,28 +0,0 @@
-name: release-create-kpt-functions
```

Comment:
This GH actions is for releasing npm module https://www.npmjs.com/package/create-kpt-functions. It does the scaffolding for kpt functions.
I guess we should not remove unless you want to re-work the TS modules release pipeline.